1080i/Includes_Labels.xml

```diff
@@ -31,6 +31,12 @@
         <value condition="$EXP[Exp_IsPVRItem] + !String.IsEmpty(ListItem.Title)">$INFO[ListItem.Title]</value>
     </variable>
 
+    <variable name="Label_Setting_HomeMenuUp">
+        <value condition="String.IsEmpty(Skin.String(HomeMenu.OnUp))">$LOCALIZE[31290]</value>
+        <value condition="Skin.String(HomeMenu.OnUp,Submenu)">$LOCALIZE[31291]</value>
```

Comment:
Same. These should be flipped so that it reflects default behaviour. Probably would also want to change from `Submenu` to `None` or something like that.

Reply:
Fixed. Used `None` as suggested for the do nothing option.